cpp/PdBase.hpp

```diff
@@ -24,7 +24,10 @@
 
 // define this to use C++11 std::mutex for locking
 #ifdef LIBPD_USE_STD_MUTEX
-    #if __cplusplus < 201103L
+	#ifdef _WIN32
+		// __cplusplus is badly behaved in windows. just assume we have mutex
```

Comment:
The problem is that some versions of msvc implements mutex but have _cplusplus=199711L. I agree it is bad to assume. One way around this would be to use _MSC_VER or a user-configurable header.